unpack.js

```diff
@@ -49,10 +49,10 @@ function extractTarball() {
   input
     .on("error", log)
     .pipe(zlib.Unzip())
-    .pipe(tar.Extract({ path: __dirname }))
+    .pipe(tar.x({ C: __dirname }))
```

Comment:
@mayrbenjamin92 Can you tell me what happens on a Windows system? I've limited access to it. As far as I can tell, this looks like it ought to work, although I'd use `cwd:` rather than `C:` and I'd put in a fallback to `tar` < 3.0 so old installs work right.